### PR TITLE
readme and contributing updates / typo fixes

### PR DESCRIPTION
**Pull request by zekiel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Welcome to DuckDuckHack's Goodies Instant Answers Repository
+# Welcome to DuckDuckHack's Spice Instant Answers Repository
```
Comment (moollaza) [inline README.md]: I think these were intended for the Spice repo?

**Comment by talsraviv** [comment]
@moollaza @zekiel I'm confused - any action items for me?

**Comment by moollaza** [comment]
@talsraviv I was /cc-ing you for your to give any extra feedback.

@zekiel was this intended for Spice or goodies?

**Comment by GuiltyDolphin** [comment]
@zekiel Situation?

You've got some *merge conflicts!*

**Comment by GuiltyDolphin** [comment]
@zekiel The version over on Spice has been merged - if we get this up-to-date with `master` (and fix Spice->Goodie typos) we can get this even with Spice!

**Comment by zekiel** [comment]
I'm just gonna close it -- Looks like a copy/paste fail here as well :) Thanks for the spot @GuiltyDolphin  I'll compile a few updates this week and get to this sooner than later

**Pull request closed without merging**